### DISCUSSION TITLE
Can not find spring cloud gateway2.2.x plugin jar file in agent package.

### DISCUSSION THREAD

**Discussion by sweat123** [apache/skywalking#7613] [Q&A]
In agent document, skywalking support spring cloud gateway 2.2.x, but I can not find spring cloud gateway 2.2.x plugin in package which download from official site. 

skywalking version: 8.4

[![hUUN7Q.png](https://z3.ax1x.com/2021/08/31/hUUN7Q.png)](https://imgtu.com/i/hUUN7Q)

**Comment by sweat123** [comment]
So move which apm-spring-cloud-gateway version file into plugin package?

**Comment by wu-sheng** [comment]
In our tests, 2.1 and 2.2 share the same code bases. 

https://github.com/apache/skywalking-java/blob/d47f9cc370ddeefbe13b188e726423e6498863c2/test/plugin/scenarios/gateway-2.1.x-scenario/support-version.list#L17-L18
